OsmAnd/src/net/osmand/plus/views/PointImageUtils.java

```diff
@@ -33,6 +34,31 @@ public class PointImageUtils {
 
 	private static final TreeMap<String, PointImageDrawable> DRAWABLE_CACHE = new TreeMap<>();
 
+	private static final Paint BITMAP_PAINT = createBitmapPaint();
```

Comment:
1 - revert all changes to PointImageUtils. Leave it as before.
2 - move new code to NearbyPlacesTileProvider
3 - are you sure BITMAP_PAINT could be static? Is it any chance for concurrency?